.sail/Dockerfile

```diff
@@ -8,7 +8,6 @@ RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | b
 
 LABEL project_root "~/go/src/go.coder.com"
 
-# Modules break much of Go's tooling.
```

Comment:
We should add this comment back before merging

Reply:
Completely forgot about that... Will do it right now